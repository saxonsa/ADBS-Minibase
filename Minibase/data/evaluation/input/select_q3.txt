Q(x,y,z) :- R(x, y, z), x < y, z != 'rl'