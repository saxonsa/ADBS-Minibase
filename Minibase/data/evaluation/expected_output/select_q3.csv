1, 9, 'adbs'
2, 7, 'anlp'
8, 9, 'ppls'
